<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimplePatternWizardWidgetUI</class>
 <widget class="QWidget" name="SimplePatternWizardWidgetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>177</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <widget class="KLineEdit" name="klineedit_pattern"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_album_example">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>21</height>
      </size>
     </property>
     <property name="text">
      <string>Example:</string>
     </property>
     <property name="buddy">
      <cstring>klineedit_example</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="KLineEdit" name="klineedit_example">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>224</red>
           <green>223</green>
           <blue>223</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>65</red>
           <green>139</green>
           <blue>212</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="HighlightedText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>224</red>
           <green>223</green>
           <blue>223</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>65</red>
           <green>139</green>
           <blue>212</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="HighlightedText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>244</red>
           <green>244</green>
           <blue>244</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>65</red>
           <green>139</green>
           <blue>212</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="HighlightedText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KUrlLabel" name="kurllabel_aboutschemes">
       <property name="text">
        <string>About schemes</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KUrlLabel" name="kurllabel_aboutparameters">
       <property name="text">
        <string>About parameters</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="KPushButton" name="kpushbutton_albumartist">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Album Artist</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="KPushButton" name="kpushbutton_albumtitle">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Album Title</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="KPushButton" name="kpushbutton_genre">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Genre</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="KPushButton" name="kpushbutton_cdno">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>CD #</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="KPushButton" name="kpushbutton_date">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="KPushButton" name="kpushbutton_suffix">
     <property name="text">
      <string>Suffix</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="KPushButton" name="kpushbutton_nooftracks">
     <property name="text">
      <string># of Tracks</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>14</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlLabel</class>
   <extends>QLabel</extends>
   <header>kurllabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
